packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -126,7 +126,9 @@ export class AnalyticsModel {
         return {
             numberUsers: usersInProject.length,
             numberViewers: usersInProject.filter(
-                (user) => user.role === OrganizationMemberRole.VIEWER,
+                (user) =>
```

Comment:
Makes sense, we can open a separate ticket if we want to improve the analytics dashboard.